src/ngx_http_lua_upstream_module.c

```diff
@@ -98,6 +99,9 @@ ngx_http_lua_upstream_create_module(lua_State * L)
     lua_pushcfunction(L, ngx_http_lua_upstream_set_peer_down);
     lua_setfield(L, -2, "set_peer_down");
 
+    lua_pushcfunction(L, ngx_http_lua_upstream_get_next_peer);
+    lua_setfield(L, -2, "get_next_peer");
+
```

Comment:
Hmm, seems like we also need to update the initial table size in the `lua_createtable()` above to avoid table auto-growth as well :)